jarbas/chamber_of_deputies/tests/test_reimbursement_view.py

```diff
@@ -1,8 +1,13 @@
+import sys
+
 from json import loads
 from unittest.mock import patch
 from urllib.parse import urlencode
 
+from django.core.management import call_command
 from django.shortcuts import resolve_url
+from django.utils.six import StringIO
+
```

Comment:
I see no need for this blank line separating one block of third-party imports from other third-party imports. What do you think about it?